etc/CMakeLists.txt

```diff
@@ -12,3 +12,51 @@ install(
     DESTINATION
         etc
 )
+
+if(NOT EXISTS ${CMAKE_INSTALL_PREFIX}/etc/nebula-graphd.conf)
+    install(
+        FILES
+        nebula-graphd.conf.default
+        PERMISSIONS
+            OWNER_READ
+            OWNER_WRITE
+            GROUP_READ
+            WORLD_READ
+        DESTINATION
+        etc
+        RENAME
+        nebula-graphd.conf
+    )
+endif()
+
+if(NOT EXISTS ${CMAKE_INSTALL_PREFIX}/etc/nebula-storaged.conf)
+    install(
+        FILES
+            nebula-storaged.conf.default
+        PERMISSIONS
+            OWNER_READ
+            OWNER_WRITE
+            GROUP_READ
+            WORLD_READ
+        DESTINATION
+            etc
+        RENAME
+            nebula-storaged.conf
+    )
+endif()
+
+if(NOT EXISTS ${CMAKE_INSTALL_PREFIX}/etc/nebula-metad.conf)
+    install(
+        FILES
+            nebula-metad.conf.default
+        PERMISSIONS
+            OWNER_READ
+            OWNER_WRITE
+            GROUP_READ
+            WORLD_READ
+        DESTINATION
+            etc
+        RENAME
+            nebula-metad.conf
+    )
+endif()
```

Comment:
why not define a macro or function to resue the same logic?

Reply:
Thanks.